pkg/common/telemetry/names.go

```diff
@@ -435,6 +435,9 @@ const (
 	// PreferredServiceName tags the preferred service name
 	PreferredServiceName = "preferred_service_name"
 
+	// Selectors tags some group of registration selector
+	PreviouslyAttestedSelectors = "previously_attested_selectors"
```

Comment:
Why not just use the [Selectors label](https://github.com/spiffe/spire/blob/main/pkg/common/telemetry/names.go#L510-L511) ("selectors")? I don't see the need for this customizaiton. It doesn't hurt per se, just seems unnecessary (and adds 20 additional characters per instance).

Reply:
This was suggested on related issue to this a while ago. Discussing it again today, we do think that it would be less confusing overall to use "selectors" or "discovered_selectors" as we do in other places. I'll update the patch with that.